style.css

```
@@ -0,0 +1,195 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+
+a, 
+p,
+h1,
+h2,
+button {
+    font-family: "Open Sans", sans-serif;
+}
+
+h1 {
+    padding-top: 12px;
+    padding-bottom: 50px;
+
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 65px;
+
+    border-top: solid;
+    border-color: #828282;
+}
+
+h2{
+    font-size: 34px;
+    font-weight: 700;
+    line-height: 46px;
+    padding-bottom: 50px;
+}
+
+p,
+a {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24.5px;
+}
+
+
+p,
+h2 {
+    text-align: center;
+}
+
+p {
+    padding-bottom: 30px;
+}
+
+p:last-child {
+    padding-bottom: 0;
+}
+
+
+header {
+    margin-top: 25px;
+    margin-bottom: 25px;
+}
+
+.nav-list {
+    margin: 0 auto;
+    width: 50%;
+    text-align: center;
+}
+
+.nav-list li {
+    padding-right: 40px;
+}
+
+.nav-list li:last-child {
+    padding-right: 0;
+}
+
+.nav-list > li > a {
+    text-decoration: none;
+    color: #828282;
+}
+
+section {
+    padding-bottom: 100px;
+    padding-top: 100px;
+}
+
+section:first-child {
+    padding-bottom: 0;
+    padding-top: 0;
+}
+
+section:nth-child(even){
+    background-color: #F6F6F6;
+
+}
+
+.skills > h2 {
+    padding-bottom: 20px;
+}
+
+.skills > p {
+    padding-bottom: 50px;
+}
+
+li {
+    list-style: none;
+    display: inline-block;
+}
+
+.list-skills {
+    width: 100%;
+}
+
+
+.list-skills > li {
+    padding-right: 90px;
+}
+
+.list-skills li:last-child {
+    padding-right: 0;
+}
+
+
+.content {
+    margin: 0 auto;
+    width: 970px;
+}
+
+.img-DN {
+    display: block;
+    margin: 0;
+    width: 100%;
+}
+
+.a-portfolio {
+    display: block;
+    color: #070707;
+    text-align: center;
+}
+
+.container-portfolio {
+    padding-bottom: 70px;
+}
+
+.container-portfolio:last-child {
+    padding-bottom: 0;
+}
+
+.container-portfolio > img {
+    padding-bottom: 50px;
+}
+
+footer {
+    text-align: center;
+    padding-top: 100px;
+    padding-bottom: 100px;
+}
+
+.text-contacts > h2 {
+    padding-bottom: 20px;
+}
+
+.text-contacts {
+    padding-bottom: 60px;
+}
+
+.cm {
+    padding-bottom: 60px;
+}
+
+.list-cm > li {
+    padding-right: 50px;
+}
+
+.list-cm > li:last-child {
+    padding-right: 0;
+}
+
+button {
+    background: #070707;
+    color: #FFFFFF;
+    border-radius: 22px;
+    padding: 10px 16px;
+    margin-bottom: 30px;
+}
+
+.p-last-m {
+    font-size: 14px;
+    color: #828282;
+}
+
+
```

```suggestion
img {
    width: 100%;
    height: auto;
}
```



https://www.w3schools.com/howto/howto_css_image_responsive.asp